debezium-connect-rest-extension/src/main/java/io/debezium/kcrestextension/entities/TransformDefinition.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.kcrestextension.entities;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.runtime.isolation.PluginDesc;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * JSON model that describes a Single Message Transform (SMT) entry.
+ */
+public class TransformDefinition extends PluginDefinition {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TransformDefinition.class);
+
+    @JsonCreator
+    public TransformDefinition(String className, ConfigDef config) {
+        super(className, config);
+    }
+
+    @JsonCreator
+    public static TransformDefinition fromPluginDesc(PluginDesc<Transformation<?>> transformPlugin) {
```

Comment:
See my other comment. Isn't a change like this too huge for this PR and for doing it now? I am not convinced that it is good architecture (although it can be a good architecture) to adapter (more or less 1:1) to Kafka classes when I look at the stability and compatibility over time. It should be worth to introduce that complexity and effort, but I am not yet convinced (also I generally understand and promote such architecture - when it is worth the effort). But change my mind! :wink: